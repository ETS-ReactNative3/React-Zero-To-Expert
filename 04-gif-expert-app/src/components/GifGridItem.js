import React from 'react'

const GifGridItem = ({ id, title, url }) => {
  // console.log(img)
  console.log(id, title, url)
  return (
    <>
      <img src={url} alt={title} />
      <p>{title}</p>
    </>
  )
}

export default GifGridItem
