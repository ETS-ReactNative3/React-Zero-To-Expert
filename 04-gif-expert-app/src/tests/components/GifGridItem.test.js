import React from 'react'
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import GifGridItem from '../../components/GifGridItem'

describe('Pruebas en <GifGridItem.js />', () => {
  test('debe de mostrar el componente correctamente', () => {
    let wrapper = shallow(<GifGridItem />)
    expect(wrapper).toMatchSnapshot()
  })
})
