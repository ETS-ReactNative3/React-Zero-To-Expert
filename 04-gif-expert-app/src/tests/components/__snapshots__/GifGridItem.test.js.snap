// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <GifGridItem.js /> debe de mostrar el componente correctamente 1`] = `
<div
  className="card animate__animated animate__flip"
>
  <img />
  <p />
</div>
`;
